README.md

```diff
@@ -34,3 +34,11 @@ Running, testing, and administering this app follows the same patterns as our ot
 *Note: for js tests, run the commands in `frontend/public`, or run via `yarn workspaces foreach run <command>` from the root.
 
 [See the common OL web app guide](http://mitodl.github.io/handbook/common-web-app-guide.html#running-and-accessing-the-app).
+
+
+# Certificates
+
+In order to manage the certificates, follow these steps:
+
+* Create a template (In CMS, go to a course page and add a certificate child page). Note that this is course based which means you need to create separate templates for each course
```

Comment:
I would just add an additional step to first create a signatory in CMS before going to create the certificate page (I think the signatory is required for the certificate page).

Reply:
Oh yes, Thanks for pointing this out, I addressed the feedback, Would you like to take another look? @collinpreston

Reply:
@arslanashraf7 , looks good to me!